syntax = "proto3";

import "log.proto";

message StreamPacket {
  oneof payload {
    LogEntry log = 1;
    // Add more types here
  }
}
